src/App.tsx

```diff
@@ -30,11 +30,12 @@ function App() {
 
   // STATE - User inputs and generated cocktail
   const [mood, setMood] = useState("");
-  const [selectedIngredients, setSelectedIngredients] = useState<string[]>([]);
+  const [selectedIngredients, setSelectedIngredients] = useState<string[]>(DEFAULT_INGREDIENTS);
```

Comment:
less user input needed: assume users will have default ingredients so they can remove the ones they dont have